### DISCUSSION TITLE
Dendron vault conversion

### DISCUSSION THREAD

**Discussion by joehays** [nvim-telekasten/telekasten.nvim#350] [Q&A]
Is there a way to convert a Dendron (or Obsidian) vault to Telekasten?

**Comment by lambtho12** [comment]
You can setup vaults in telekasten if that is what you want.

I am not sure about conversion and how stuff are made in other software though. Unortunately, I barely have time to approve PRs and fix pressing issues, so if you want some specific tool for interfacing with other systems I am afraid you will have to make it yourself.
